pebb.so/include/action.h

```diff
@@ -93,6 +93,14 @@ class pebb_action : public rvs::actionbase {
   //! link type
   int link_type;
   std::string link_type_string;
+
+  //! Number of warm calls (transfer iterations) before bandwidth calculation
```

Comment:
a bit more detail on what is warm call, the reason for warm and hot calls, comments can state intent clearly